packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -88,6 +88,12 @@ export async function fetchSnapshotWithRedeem(
     removeEntries: () => Promise<void>,
     enableRedeemFallback?: boolean,
 ): Promise<ISnapshotContents> {
+    // back-compat: This block to be removed with #8784 when we only consume/consider odsp resolvers that are 0.51+
```

Comment:
Is it 0.51>= or 0.51>. 
Actually the format is //back-compat last-version-to-back-compat(maybe 0.51 in this case) <comment>

Reply:
Clarified versioning dependency.